pages/api/select.md

```diff
@@ -10,7 +10,7 @@
 | MenuProps | Object |  | Properties applied to the `Menu` element. |
 | <span style="color: #31a148">children *</span> | ChildrenArray |  | The option elements to populate the select with. Can be some `MenuItem` when `native` is false and `option` when `native` is true. |
 | classes | Object |  | Useful to extend the style applied to components. |
-| input | Element | <Input /> | An `Input` element. |
+| input | Element | &lt;Input /> | An `Input` element. |
```

Comment:
This is automatically generated, but follow a pattern close can help.
https://github.com/callemall/material-ui/blob/353f64d896eca71399c1e97a5c02d3ed5d78050c/docs/src/modules/utils/generateMarkdown.js#L42

Reply:
I'll take care of it when [parsing the default value](https://github.com/callemall/material-ui/blob/353f64d896eca71399c1e97a5c02d3ed5d78050c/docs/src/modules/utils/generateMarkdown.js#L163)